.github/workflows/build_push_publish.yml

```diff
@@ -1,11 +1,12 @@
+# Deprecated
 name: Build, push, publish CAS
 
 on:
-  push:
-    branches:
-      - main
-      - "release-candidate"
-      - develop
+#  push:
+#    branches:
+#      - main
+#      - "release-candidate"
+#      - develop
```

Comment:
We're commenting these out because we don't want this new workflow to run yet?

Reply:
This is actually the existing workflow that my changes will now make redundant. Didn't want to delete it yet, so I just disabled the automatic execution.